setup.cfg

```
@@ -59,7 +59,7 @@ zip_safe = False
 
 # These are required in actual runtime:
 install_requires =
-    ansible-compat >= 0.5.0
+    ansible-compat >= 1.0.0a0
```

Let's be stricter until the API is stable:
```suggestion
    ansible-compat ~= 1.0.0a0
```